File: 07. Cyclically rotate an array by one/rotate.py
size = int(input("Enter the nunmber of elements: \n"))

listOfElementsA = list(map(int,input("Enter the elements separated by space: \n").strip().split()))

rotationCount = int(input("Enter the roation count: \n"))    

while(rotationCount != 0):
    last = listOfElementsA[size-1]
    for i in range (size-1,0,-1):
        listOfElementsA[i] = listOfElementsA[i-1]
    listOfElementsA[0] = last
    rotationCount-=1

print(listOfElementsA)


    

File: 07. Cyclically rotate an array by one/rotate.java
import java.io.*;
import java.util.Arrays;

class rotate
{
    public static void main(String args[])throws IOException
    {
        BufferedReader br =new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Enter the number of Elements in array A:");
        int sizeA = Integer.parseInt(br.readLine());
        int listOfElementsA[] = new int[sizeA];
        System.out.println("Enter the Elements for A:");     
        
        String[] strs = br.readLine().trim().split("\\s+");
      
        for(int i=0;i<sizeA;i++)
        {
            listOfElementsA[i] = Integer.parseInt(strs[i]);
        }

        System.out.println("Enter the rotation count: ");     
        int rotationCount = Integer.parseInt(br.readLine());
        while(rotationCount!=0)
        {
            int last = listOfElementsA[sizeA-1];
            for(int i = sizeA-1;i>=1;i--)
            {
                listOfElementsA[i] = listOfElementsA[i-1];
            }
            listOfElementsA[0] = last;
            rotationCount--;
        }
        for(int i=0;i<sizeA;i++)
        System.out.print(listOfElementsA[i]+" ");


    }
}